openlibrary/catalog/merge/test_merge_marc.py

```diff
@@ -114,25 +129,29 @@ def test_build_marc():
     # used in openlibrary.catalog.add_book.load()
     # when trying to find an existing edition match
     edition = {
-            'title': 'A test title (parens)',
-            'full_title': 'A test full title : subtitle (parens).',  # required, and set by add_book.load()
+            'title': 'A test titlé (parens)',
+            'full_title': 'A test full titlé : sübtitle (parens).',  # required, and set by add_book.load()
             'source_records': ['ia:test-source']
             }
     result = build_marc(edition)
     assert isinstance(result['titles'], list)
     assert result['isbn'] == []
-    assert result['normalized_title'] == 'a test full title subtitle (parens)'
-    assert result['short_title'] == 'a test full title subtitl'
+    assert result['normalized_title'] == 'a test full titlé sübtitle (parens)'
+    assert result['short_title'] == 'a test full titlé sübtitl'
 
 
 def test_compare_publisher():
     foo = {'publishers': ['foo']}
     bar = {'publishers': ['bar']}
+    unicode_foo = {'publishers': ['føo']}
+    unicode_bar = {'publishers': ['bär']}
     foo2 = {'publishers': ['foo']}
     both = {'publishers': ['foo', 'bar']}
     assert compare_publisher({}, {}) == ('publisher', 'either missing', 0)
     assert compare_publisher(foo, {}) == ('publisher', 'either missing', 0)
     assert compare_publisher({}, bar) == ('publisher', 'either missing', 0)
+    assert compare_publisher(foo, unicode_foo) == ('publisher', 'mismatch', -25)
```

Comment:
I don't think this is correct, I think in some way the original purpose was to match on `foo` == `føo`
to account for variations in catalog entry. 

A real example:
https://openlibrary.org/search?q=publisher%3AS%C3%BCddeutscher+Verlag&mode=everything

vs.

https://openlibrary.org/search?q=publisher%3ASuddeutscher+Verlag&mode=everything

Shows that the publisher Süddeutscher Verlag is also catalogued (at least in our data, possibly elsewhere)  as Suddeutscher Verlag.

The original code I believe was actively trying to account for this and accept the variation as a match.

Reply:
Ok, I see now that this is testing existing behaviour, not making a change to that behaviour. The behaviour still looks wrong to me, but it's not the test's fault :D 
I've tried to cover this in my final review comment.